packages/price-exporter/bin/price-exporter.js

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+
+/* eslint-disable */
```

Comment:
what do we want to disable here?
if it is about the max-len on L5, this can be easily fixed. There is no need to disable eslint, however trivial the need